src/main/resources/common-services/CDAP/5.0.0/package/scripts/master.py

```diff
@@ -64,21 +64,21 @@ def killOlderProcesses(self):
         # execute ls /cdap/election/master.services and check if emty then only execute the following steps else exit
 
         import params
-        Execute("/usr/hdp/3.1.0.0-78/zookeeper/bin/zkCli.sh -server " + params.cdap_zookeeper_quorum + " ls /election/master.services > /tmp/election_master_services", user='zookeeper')
+        Execute("/usr/hdp/current/zookeeper-client/bin/zkCli.sh -server " + params.cdap_zookeeper_quorum + " ls /election/master.services > /tmp/election_master_services", user='zookeeper')
```

Comment:
Add a constant variable to store value of path '/usr/hdp/current/zookeeper-client/bin/zkCli.sh' and use this variable everywhere in the code.

Reply:
Created a variable for the value '/usr/hdp/current/zookeeper-client/bin/zkCli.sh -server ' + params.cdap_zookeeper_quorum